brainbeats-backend/Controllers/BeatController.cs

```diff
@@ -1,39 +1,49 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Newtonsoft.Json.Linq;
 using static brainbeats_backend.QueryStrings;
 using static brainbeats_backend.Utility;
 
 namespace brainbeats_backend.Controllers {
+  public class Beat {

```

Comment:
Can we investigate a code agnostic way of defining these schemas? I imagine if this needs to be updated it needs to be changed in nearly 3 different repos